src/pages/Pages.tsx

```diff
@@ -0,0 +1,32 @@
+import { lazy, Suspense } from "react";
+import { Route, Routes } from "react-router-dom";
+
+const Home = lazy(async () => await import("./home/home"));
```

Comment:
Nice! I think you can drop the `async` and `await` here and just do something like:

```
const Home = lazy(() => import("./home/home"))
```